pkg/isbsvc/jetstream_service.go

```diff
@@ -21,6 +25,21 @@ type jetStreamSvc struct {
 	js       *jsclient.JetStreamContext
 }
 
+func (jss *jetStreamSvc) CreateWatermarkFetcher(ctx context.Context, pipelineName string, bufferName string) (fetch.Fetcher, error) {
```

Comment:
`pipelineName` is already a property of the struct, not needed in the function signature.

Reply:
Thanks! Addressed.